generator/translators_pool.cpp

```diff
@@ -50,9 +50,19 @@ bool TranslatorsPool::Finish()
     std::future<TranslatorPtr> right;
     queue.WaitAndPop(left);
     queue.WaitAndPop(right);
-    queue.Push(pool.Submit([left{std::move(left)}, right{std::move(right)}]() mutable {
```

Comment:
Curious, what was the build error here?

Reply:
It's in the commit message: https://developercommunity.visualstudio.com/t/108672

Reply:
Thanks! If comments with links were added into the code, I (and nobody else) wouldn't miss it )